imager_profile/tests/test_profile_view.py

```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase
+from django.urls import reverse
+from imagersite.tests.factories import ValidatedUserFactory
+
+
+class ProfileViewNotLoggedInTestCase(TestCase):
```

Comment:
Classes and functions in this module all need docstrings.

All docstrings should be triple double quotes, not triple single quotes.